### PR TITLE
Update samples to 2018.3

### PR DESCRIPTION
**Pull request by tdmowrer**
This updates the samples repo to be based on Unity 2018.3.0f1. Other improvements:

- Update the README.md (and also cleanup awkward language and correct bad grammar).
- Add public getters/setters for `[SerializeField]` fields which caused several warnings in the Editor if using C# 4.0
- Minor code cleanup

### CODE REVIEW

**Comment by tdmowrer** [comment]
Closed by accident. Reopening.

File: Packages/manifest.json [hunk 1]
```diff
@@ -1,7 +1,43 @@
...
+    "com.unity.modules.unitywebrequest": "1.0.0",
+    "com.unity.modules.unitywebrequestassetbundle": "1.0.0",
+    "com.unity.modules.unitywebrequestaudio": "1.0.0",
+    "com.unity.modules.unitywebrequesttexture": "1.0.0",
+    "com.unity.modules.unitywebrequestwww": "1.0.0",
+    "com.unity.modules.vehicles": "1.0.0",
+    "com.unity.modules.video": "1.0.0",
+    "com.unity.modules.vr": "1.0.0",
+    "com.unity.modules.wind": "1.0.0",
+    "com.unity.modules.xr": "1.0.0"
```
Comment (mdurand42) [inline Packages/manifest.json]: We have `com.unity.modules.vr` and `com.unity.modules.xr`? I'll look into why.

**Comment by tdmowrer** [comment]
> We should wait until 2018.3 release is available to the public.

Definitely.

**Comment by mdurand42** [comment]
Not sure when 2018.3 will be released. Was supposed to be today but there is a one `shipstopper` open against it.

**Pull request merged**